### DISCUSSION TITLE
A Coding Companion that stopped learning 3 years ago?!

### DISCUSSION THREAD

**Discussion by CAVALICIOUSDEV** [community/community#113929] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I've recently started learning tailwind and since it's a huge library, I thought that CoPilot would be perfect. Although now I see that it doesn't include the 3.0 and beyond update, which makes it useless! How is it acceptable to be paying 10 USD a month for an AI companion that doesn't update itself with knowledge past September 2021? Especially when ChatGPT is free and is also based on information garnered up-to 2021 Sep. 

This is totally unacceptable in my opinion, anyway I'm not paying for this ChatGPT copy anymore. Just wanted to warn you!

**Comment by vickkie** [comment]
Hey @CAVALICIOUSDEV 

 Understandable.

**Comment by vasubhalodiya** [comment]
Hi
